cmd/juju/action/exec_test.go

```diff
@@ -0,0 +1,656 @@
+// Copyright 2013 Canonical Ltd.
```

Comment:
moved from cmd/juju/commands

Reply:
> LGTM. One nit comment. Also a question: why does the facade version not have to be incremented due to the facade return value changes?

This is for juju 3 and we are disallowing version 2 clients from using this api. You will need a v 3 client.